content/docs/instrumenting/exporters.md

```diff
@@ -38,6 +38,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
    * [Druid Exporter](https://github.com/opstree/druid-exporter)
    * [ElasticSearch exporter](https://github.com/justwatchcom/elasticsearch_exporter)
    * [EventStore exporter](https://github.com/marcinbudny/eventstore_exporter)
+   * [KDB+ exporter](https://github.com/KxSystems/prometheus-kdb-exporter) (**official**)
```

Comment:
This is not an official exporter.

Reply:
Does official mean that its an official exporter that's from Prometheus, or the official exporter from the product that's it is to be used with?
I had interpreted that its official if its created by the company upon which its used.

Thanks

Reply:
Sorry, just noticed on the page it had 
'Some of these exporters are maintained as part of the official Prometheus GitHub organization, those are marked as official, others are externally contributed and maintained.'
Will change it.